_partials/_devops-cli-global-flags.md

```diff
@@ -0,0 +1,10 @@
+
+| Flag                      | Default         | Description                                                                                                                                                                                                                 |
+|---------------------------|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--analytics`               | `true`          | Set to `false` to disable usage analytics.                                                                                                                                                                                  |
+| `--config-dir` string       | `.config/tiger` | Set the directory that holds `config.yaml`                                                                                                                                                                                  |
```

Comment:
Could you either remove all periods at the end of the sentences, or have them everywhere in this table.